src/api/app/controllers/webui/requests/main_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Webui
+  module Requests
+    class MainController < WebuiController
+      protected
+
+      def request_action_attributes(type)
```

Comment:
This needs to be replaced by strong parameters with nested attributes for the `bs_request_actions` association in the `BsRequest` model.

Reply:
This file was removed.